sheet_to_triples/tests/test_rdf.py

```diff
@@ -312,3 +264,109 @@ def test_graph_from_triples(self):
             with self.subTest(value=expected_value):
                 self.assertIsInstance(test_value, instance)
                 self.assertEqual(str(test_value), expected_value)
+
+
+class TestResolver(unittest.TestCase):
+
+    @property
+    def graph(self):
+        return rdf._new_graph()
+
+    def test_from_identifier_known_iri_qname_returns_uriref(self):
+        tests = {
+            ('vm:Category', 'http://visual-meaning.com/rdf/Category'),
+            ('gist:isCategorizedBy',
+                 'https://ontologies.semanticarts.com/gist/isCategorizedBy'),
+            ('rdf:type', 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type'),
+        }
+        for value, expected in tests:
+            with self.subTest(value=value):
+                resolver = rdf.Resolver(self.graph)
+                self.assertEqual(
+                    resolver.from_identifier(value),
+                    rdflib.term.URIRef(expected),
+                )
+
+    def test_from_identifier_known_iri_url_returns_uriref(self):
+        tests = {
+            'http://visual-meaning.com/rdf/Category',
+            'https://ontologies.semanticarts.com/gist/isCategorizedBy',
+            'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',
+        }
+        for value in tests:
+            with self.subTest(value=value):
+                resolver = rdf.Resolver(self.graph)
+                self.assertEqual(
+                    resolver.from_identifier(value),
+                    rdflib.term.URIRef(value),
+                )
+
+    def test_from_identifier_unknown_url_returns_literal(self):
```

Comment:
This test seems needlessly forward compatible, with a loop over one item in a set and subtest - ideally subtest values are short too.